### PR TITLE
feat: add hello world to module 3

### PR DESCRIPTION
**Pull request by jamesrp13**

### CODE REVIEW

**Review by mixelpixel — APPROVED** [review]
Hello, world! looks good

**Comment by jamesrp13** [comment]
> @jamesrp13 noting here per the Movie Review/Basic Program Pt 1 PR [#64 (comment)](https://github.com/Unboxed-Software/solana-course/pull/64#discussion_r896248656), I think we might want to add an "iterator" section to the Hello World Program lesson?
> 
> ```
>     // Iterators
>     let v1 = vec![1, 2, 3];
>     /* create the iterator over the vector: */
>     let mut v1_iter = v1.iter();
>     /* use the iterator to get the first item */
>     let first_item = v1_iter.next();
> ```

Not sure what I wrote in the comment, but what I meant was to add iterator section to the movie review pt 2 lesson, not the hello world lesson

**Review by ZYJLiu — APPROVED** [review]

File: content/hello-world-program.md [hunk 1]
```diff
@@ -30,23 +30,29 @@ Rust organizes code using what is collectively referred to as the “module syst
...
 
-1. Bring the module into the scope of our program
-2. Specify the item in the module tree using its path
+The path to a particular module or item is the name of each step from the crate to that module where each is separated by `::`. As an example, let's look at the following structure:
 
-Paths are brought into scope with the [use](https://doc.rust-lang.org/stable/book/ch07-04-bringing-paths-into-scope-with-the-use-keyword.html) keyword. In the example below, we bring into scope the `AccountInfo` struct from the `account_info` module within the `solana_program` crate. Bringing an item into scope this way allows us to use the item multiple times within our program without specifying the full path each time.
+1. The base crate is `solana_program`
+2. `solana_program` contains a module named `account_info`
+3. `account_info` contains a struct named `AccountInfo`
+
```
Comment (mixelpixel) [inline content/hello-world-program.md:50]: weird, my text editor shows two weird little orange dots here... not sure what they are but jsyk see pic:
pr61-weirdOrangeDots

File: content/hello-world-program.md [hunk 2]
```diff
@@ -30,23 +30,29 @@ Rust organizes code using what is collectively referred to as the “module syst
...
-Paths are brought into scope with the [use](https://doc.rust-lang.org/stable/book/ch07-04-bringing-paths-into-scope-with-the-use-keyword.html) keyword. In the example below, we bring into scope the `AccountInfo` struct from the `account_info` module within the `solana_program` crate. Bringing an item into scope this way allows us to use the item multiple times within our program without specifying the full path each time.
+1. The base crate is `solana_program`
+2. `solana_program` contains a module named `account_info`
+3. `account_info` contains a struct named `AccountInfo`
+   
+The path to `AccountInfo` would be `solana_program::account_info::AccountInfo`. 
+
+Absent of any other keywords, we would need to reference this entire path to use `AccountInfo` in our code.
+
+However, with the [`use`](https://doc.rust-lang.org/stable/book/ch07-04-bringing-paths-into-scope-with-the-use-keyword.html) we can bring an item into scope so that it can be reused throughout a file without specifying the full path each time. It's common to see a series of `use` commands at the top of a Rust file.
```
Comment (mixelpixel) [inline content/hello-world-program.md:55]: fix: awk 
add "keyword" --> with the `use` keyword we can bring

**Review by mixelpixel — CHANGES_REQUESTED** [review]
1. optional: remove weird dots
2. fix: `use` keyword

**Pull request merged**